### DISCUSSION TITLE
Internal resolution scale

### DISCUSSION THREAD

**Discussion by chrism69** [emoose/OutRun2006Tweaks#260] [Suggestions]
### Describe the feature you'd like to see added

Like most new games, I'd like an option to run in desktop resolution but scale down the internal resolution to match the original artwork, without running the whole game at that resolution with all the Windows scaling problems that brings.

**Comment by Outrunner24** [comment]
I may be wrong but you can't do this without going deep into the game code and how it's using the graphics driver/DirectX interface. 

Also, there's already an .INI option for DPI that overrides Windows' settings.

**Comment by brad86m** [comment]
If you use Nvidia Profile Inspector, you can force SGSSAA. This is the best form of downsampling and AA that has ever existed. Far, far better than anything with have now, like DSR. Sadly, it's old and doesn't really work post DX9, but the good news is that OutRun is DX9 :)
